benchmarks/timeseries/periodic_feature_extractor.py

```diff
@@ -34,7 +35,7 @@ async def init_feature_extractor(period: int) -> PeriodicFeatureExtractor:
         timedelta(seconds=1), lm_chan.new_receiver(), timedelta(seconds=1)
     )
 
-    await lm_chan.new_sender().send(Sample(datetime.now(tz=timezone.utc), 0))
+    await lm_chan.new_sender().send(Sample(datetime.now(tz=timezone.utc), Quantity(0)))
```

Comment:
`Quantity` is being used as a dimensionless type,  so the resampler for example would return objects of this type.

Reply:
> Is it only needed for type hinting or do you see any other use cases for it?

The reason it it like this is because in cases like the FeatureExtractor,  we need a fallback type that everything would decompose into.  So we pass any receiver to the FeatureExtractor,  and it would be treated as a `Quantity`.  Because from that point we'll only deal with numpy arrays, for example,  we can't propagate the types downstream.